frontend/src/pages/videos/VideoAnalysisPage.tsx

```diff
@@ -23,12 +33,15 @@ export const VideoAnalysis = ({
 }) => {
   const { t } = useTranslation();
   const { baseUrl } = useCurrentPoll();
+  const [expanded, setExpanded] = React.useState(false);
```

Comment:
I feel that the name `expanded` does not describe enough what is expanded or not (here the description). May be a more specific name can improve comprehension ?